src/tests/markdown.spec.js

```diff
@@ -1,7 +1,11 @@
 import spec from "./fixtures/spec"
 import markdownit from './../markdownit'
 import { typesAvailable } from './../markdownit/callouts'
-import { markdownThroughEditor, markdownThroughEditorHtml } from "./helpers";
+import {
+	markdownThroughEditor,
+	markdownThroughEditorHtml,
+	markdownFromPaste
+} from "./helpers";
```

Comment:
Nitpicking, but this probably needs adjustments to please the linter (single quotes, no semicolon).

Reply:
Good catch. Linter is not complaining though... funny.